src/elements/helpers/Preview.tsx

```diff
@@ -24,6 +47,28 @@ const getDocHeight = (doc: Document | undefined) => {
   return undefined;
 };
 
+function isUntypedMessageData(data: unknown): data is UntypedMessageData {
+  if (
+    typeof data === "object" &&
+    data !== null &&
+    "value" in data &&
+    "type" in data
+  ) {
+    return true;
+  }
+  return false;
+}
```

Comment:
This seems like a slightly unnecessary step. It might be better to incorporate this logic into `isMessageData` and remove the `UntypedMessageData` type.

Reply:
This was the result of some frustrating type shenanigans with Typescript & Eslint TS was not happy that the properties existed unless I used a type predicate .I've had another go this morning, with a Typescript friendly generalised property-finding type predicate helper (`hasOwnProperty`) - and have been able to remove `isUntypedMessageData`.